install.py

```diff
@@ -26,8 +31,18 @@ def hookpath():
 
 print()
 print("✔️  Detect-secrets hook installed")
+print()
 
-register("test")  # Remove test mode argument for production
+print(
+    "ℹ️ The Cyber Security Team would like to register your installation, "
+    + "only your github username will be securely sent to us."
```

Comment:
What we actually do is create a Personal Access Token with `read:user` and `read:org` permissions and use that token to identify the user. Don't know if you want to go into that level of detail but it might be worth saying a PAT with limited permissions will be securely sent to us.

Reply:
good point, I'll update